tools/Generator/Generator.cs

```diff
@@ -187,17 +192,9 @@ bool WriteEnum (XElement enumElement)
 
 			var enumsElement = values.First ();
 			if (enumsElement.Attribute ("type") != null && enumsElement.Attribute ("type").Value == "bitmask") {
-				string suffix = null;
-				foreach (var ext in extensions)
-					if (csName.EndsWith (ext.Key)) {
-						suffix = ext.Key + suffix;
-						csName = csName.Substring (0, csName.Length - ext.Key.Length);
-					}
+				
```

Comment:
So, I just moved the code who replace the extension suffix to the GetCsTypeName method, current code generate "CompositeAlphaFlagsKHR", my code generate "CompositeAlphaFlagsKhr".  The enum's members aren't generated right, I fixed it locally (removed the enum name prefix), I will commit it soon.